tasks/post-deploy/05-clean-deprecated-pm-key.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+# PLANET-4782
+# List existing deprecated keys
+echo "Usage of '_campaign_page_template' meta key..."
+wp db query 'SELECT wp.*, wp2.meta_key as theme_key, wp2.meta_value as theme_value
+    FROM wp_postmeta wp
+    LEFT JOIN wp_postmeta wp2 ON (
+      wp.post_id = wp2.post_id
+      AND wp2.meta_key = "theme"
+    )
+    WHERE wp.meta_key="_campaign_page_template";'
+
+# Convert deprecated keys to new keys if possible
+echo "Replacing key with 'theme'"
+wp db query 'UPDATE wp_postmeta
+    SET meta_key = "theme"
+    WHERE meta_key="_campaign_page_template"
+      AND post_id NOT IN (
```

Comment:
I can't directly use the same table used in `UPDATE` because mysql considers it locked, that's why there's a `select * from` around it; but even with that I think I can't make the condition work (unless using a variable maybe), it says it doesn't know `pm.post_id`, probably doesn't have access to the first table.